appshell/mac/Brackets.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Give preference to launching the one in /Applications folder
```

Comment:
I guess the `open -a Brackets "$@"` fallback could have that problem too since I think it just does a global Info.plist search across every app bundle on the machine.  Should we first try a relative path to the Brackets executable (since this script is in the bundle right next to it)?